tests/hie-bios/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+load(
+    "@rules_haskell//haskell:defs.bzl",
+    "haskell_repl",
+    "haskell_test",
+)
+
+haskell_test(
+    name = "binary-hie-bios-test",
+    srcs = ["Main.hs"],
+    deps = ["//tests/hackage:base"],
+)
+
+haskell_repl(
+    name = "hie-bios",
+    testonly = 1,
+    collect_data = True,
+    hie_bios_path_prefix = ["$magic_string"],
+    deps = [":binary-hie-bios-test"],
+)
+
+haskell_repl(
+    name = "hie-bios-no-prefix",
+    testonly = 1,
+    collect_data = True,
+    deps = [":binary-hie-bios-test"],
+)
+
+filegroup(
+    name = "hie-bios-file",
+    testonly = 1,
+    srcs = [":hie-bios"],
+    output_group = "hie_bios",
+)
+
+filegroup(
+    name = "hie-bios-file-no-prefix",
+    testonly = 1,
+    srcs = [":hie-bios-no-prefix"],
+    output_group = "hie_bios",
+)
+
+sh_test(
+    name = "test-hie-bios",
+    size = "small",
+    srcs = ["test_hie_bios.sh"],
+    args = [
+        "$(rootpath :hie-bios-file)",
+        "$(rootpath :hie-bios-file-no-prefix)",
+    ],
+    data = [
+        ":hie-bios-file",
+        ":hie-bios-file-no-prefix",
+    ],
+    tags = ["dont_test_on_windows"],
```

Comment:
```suggestion
```
This shouldn't be necessary. rules_haskell requires msys2 on Windows anyway, so a bash is availabl. And Bazel can generate executables from `.sh` scripts on Windows if a bash is a available.

Reply:
Just saw the [Windows CI failure](https://github.com/tweag/rules_haskell/runs/2325545499#step:6:102):
```
grep: tests/hie-bios/hie-bios@hie-bios: No such file or directory
```
I think the issue has to do with runfiles. On Unix Bazel will create a tree of symlinks in the tests working directory pointing to `data` dependencies. However, on Windows symlinks are not generally available, so instead Bazel generates a runfiles manifest file and code needs to use a runfiles library to query the manifest file for the location of `data` dependencies. The bash runfiles library is documented [here](https://github.com/bazelbuild/bazel/blob/master/tools/bash/runfiles/runfiles.bash#L39-L68). Example uses can be found in the repository. In tests you can use the `TEST_WORKSPACE` env-var to find the workspace name, e.g.
```
file1="$(rlocation "$TEST_WORKSPACE/$1")"
```

Reply:
Ah I see, thank you.

So I activated the windows test back on and it works with the bash runfiles library.
And now, debug information is only printed in case the test fails.